test/integration/model/upsert.test.js

```diff
@@ -311,6 +311,31 @@ describe(Support.getTestDialectTeaser('Model'), () => {
         clock.restore();
       });
 
+      it('falls back to a noop if no update values are found in the upsert data', async function() {
+
+        const User = this.sequelize.define('user', {
+          username: DataTypes.STRING,
+          baz: {
+            type: DataTypes.STRING,
+            field: 'zab',
```

Comment:
this is melting my brain. 🙈 Can you maybe introduce some real-world property names and values?

Reply:
switched `baz` to `email`.